src/chttpd/src/chttpd_db.erl

```diff
@@ -18,17 +18,34 @@
 -include_lib("couch_views/include/couch_views.hrl").
 -include_lib("kernel/include/logger.hrl").
 
--export([handle_request/1, handle_compact_req/2, handle_design_req/2,
-    db_req/2, couch_doc_open/4,handle_changes_req/2,
+-export([
+    handle_request/1,
+    handle_compact_req/2,
+    handle_design_req/2,
+    db_req/2,
+    couch_doc_open/4,
+    handle_changes_req/2,
     update_doc_result_to_json/1, update_doc_result_to_json/2,
-    handle_design_info_req/3, handle_view_cleanup_req/2,
-    update_doc/4, http_code_from_status/1]).
-
--import(chttpd,
-    [send_json/2,send_json/3,send_json/4,send_method_not_allowed/2,
-    start_json_response/2,send_chunk/2,end_json_response/1,
-    start_chunked_response/3, absolute_uri/2, send/2,
-    start_response_length/4]).
+    handle_design_info_req/3,
+    handle_view_cleanup_req/2,
+    update_doc/4,
+    http_code_from_status/1
+]).
+
+-import(
+    chttpd,
+    [
+        send_json/2, send_json/3, send_json/4,
```

Comment:
0. It might be better to put every function on new line. However I do acknowledge the fact that it would eat too much of precious vertical space.

Reply:
If you split the functions manually, they will be kept separate - erlfmt preserves this kind of grouping of exports with the same name to appear on the same line, but does not force it.